Components/ReusableComponent/functions.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Flynt\Components\ReusableComponent;
+
+use Timber\Timber;
+
+add_filter('Flynt/addComponentData?name=ReusableComponent', function ($data) {
+    if (!empty($data['reusableId'])) {
+        $data['post'] = Timber::get_post($data['reusableId']);
+    }
+
+    return $data;
+});
+
+function getACFLayout()
+{
+    return [
+        'name' => 'ReusableComponent',
+        'label' => 'Reusable <i class="dashicons dashicons-controls-repeat"></i>',
+        'sub_fields' => [
+            [
+                'label' => __('Select Component Set', 'flynt'),
```

Comment:
Please see my following proposal:

Select Component Set => Select Reusable Components
Add or edit reusable components. => Add reusable components.
Edit component. => Move to same line.
![Bildschirmfoto 2020-09-30 um 16 48 24](https://user-images.githubusercontent.com/102498/94706262-4842e300-0342-11eb-93ec-ccf1a36372ae.png)